### PR TITLE
add persist for clink

### PR DESCRIPTION
**Pull request by candrapersada**
add `clink --profile "$dir\\config"` or `"clink.bat --profile "$dir\\config"`

### CODE REVIEW

**Comment by rashil2000** [comment]
This was present before and was removed: https://github.com/ScoopInstaller/Main/commit/a8f0b56d0bfc1db9ae2adef98395b70cae48ded7 because it doesn't get picked up automatically.

**Comment by candrapersada** [comment]
but work if add `clink --profile "$dir\\config"` or `"clink.bat --profile "$dir\\config"` on clink.bat in shims

**Comment by candrapersada** [comment]
but work if add `clink --profile "$dir\\config"` or `"clink.bat --profile "$dir\\config"` on clink.bat in shims
```
@rem C:\Users\Candra\scoop\apps\clink\current\clink.bat
@"C:\Users\Candra\scoop\apps\clink\current\clink.bat" --profile "C:\Users\Candra\scoop\apps\clink\current\config"  %*
```

**Comment by rashil2000** [comment]
> but work if add `clink --profile "$dir\\config"` or `"clink.bat --profile "$dir\\config"` on clink.bat in shims

Yes, but we are not modifying shims here, as that would be a breaking change for users, and it will block the `--profile` flag for them.

**Comment by candrapersada** [comment]
https://chrisant996.github.io/clink/clink.html#portable-configuration
image

File: bucket/clink.json [hunk 1]
```diff
@@ -9,7 +9,11 @@
     ],
     "url": "https://github.com/chrisant996/clink/releases/download/v1.4.3/clink.1.4.3.67deea.zip",
     "hash": "dca4ea9f02da7063d76fb56d7958af6016be7f7ee01d8e78047c5a9d4fa82bdc",
-    "bin": "clink.bat",
+    "bin": [
+        "clink.bat",
+        "portable.bat"
```
Comment (rashil2000) [inline bucket/clink.json:14]: There's no file called portable.bat in the downloaded archive

**Pull request closed without merging**